File: 2011.执行操作后的变量值.java
/*
 * @lc app=leetcode.cn id=2011 lang=java
 *
 * [2011] 执行操作后的变量值
 */

// @lc code=start
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x = 0;
        for (String operation : operations) {
            if (operation.charAt(1) == '+') {
                x++;
            } else if (operation.charAt(1) == '-') {
                x--;
            }
        }
        return x;
    }
}
// @lc code=end

File: 2011.执行操作后的变量值.js
/*
 * @lc app=leetcode.cn id=2011 lang=javascript
 *
 * [2011] 执行操作后的变量值
 */

// @lc code=start
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let x = 0;
    for (const operation of operations) {
        if (operation[1] === '+') {
            x++;
        } else if (operation[1] === '-') {
            x--;
        }
    }
    return x;
};
// @lc code=end

